### DISCUSSION TITLE
Create tickets from file editor

### DISCUSSION THREAD

**Discussion by rubengmurray** [Edsol/clickup-vscode-extension#10] [Ideas]
I'd like to see a feature that enables the following 👀 

Scenario:

- I'm on a line of code where I think a bug may exists / an improvement could be made
- I don't want to action this during my piece of work as it's out of scope
- Navigating away from my editor to a browser to create the ticket is a pain, and likely too much of an ask for me to carry through
- Adding lots of comments to the code to explain the observation is overkill and bloats the code base
 
Feature:

- I right click / have a shortcut key that enables me to create a click-up ticket inside VS Code (similar already exists in this extension)
- I'm can select the space & board to raise the ticket into
  - Essentially just a couple of extra fields from the current 'add task' view
- A permalink to the line of code I'm creating the ticket from is automatically added into the description of the ticket
  - ... I add other details/thoughts/status etc
- The click-up ticket URL from the callback/response is automatically inserted into the code LN as a comment
  - e.g. // https://app.clickup.com/N/v/dc/T/2

Thoughts?

I might give this a go myself at some point, but raising here in case anyone shares the same idea and gets there before me.

**Comment by Edsol** [comment]
I just now saw your idea, sorry but I am a little busy with university and work at this time.

It's a very good idea, in brief a quickly action to create a new task. I don't know how much effort it will take to do it, I don't think it's very complex though.

I would like to continue the development, someone to help me is always welcome. 

:smile:
